### ISSUE TITLE
slack invite link does not work

### ISSUE THREAD

**Issue by athornton** [argoproj/argo#1706]
**Is this a BUG REPORT or FEATURE REQUEST?**:

BUG REPORT

**What happened**:

I clicked on the Slack Invite link, and was told it had expired.  I then tried argoproj.slack.com and was told that I should contact the workspace admin, but since no contact information was present, that didn't help.

**What you expected to happen**:

For me to be signed up for the argoproj Slack workspace, or to create a request that would be reviewed and then eventually I'd be admitted.

**How to reproduce it (as minimally and precisely as possible)**:

Go to the home page and try to join the Argoproj Slack workspace.

**Anything else we need to know?**:

**Environment**:
n/a

**Comment by simster7** [comment]
Worked for me: https://cl.ly/146d571b7eb3. Can you try again?

**Comment by athornton** [comment]
The link I see on the page is:

https://join.slack.com/t/argoproj/shared_invite/enQtMzExODU3MzIyNjYzLTA5MTFjNjI0Nzg3NzNiMDZiNmRiODM4Y2M1NWQxOGYzMzZkNTc1YWVkYTZkNzdlNmYyZjMxNWI3NjY2MDc1MzI

That yields:

<img width="1322" alt="Screen Shot 2019-10-23 at 11 41 43 AM" src="https://user-images.githubusercontent.com/2453165/67423984-249be800-f58a-11e9-872f-533839609c3d.png">

Mac OS Catalina, Firefox 69.0.3, ublock but nothing else really fancy.

**Comment by simster7** [comment]
The actual link on the readme is: https://argoproj.github.io/community/join-slack. Seems like it's only an issue on your end?

**Comment by athornton** [comment]
Odd.  It's the same both from work and home for me, so two different networks, but similar-to-identical OS/Browser combinations.

**Comment by athornton** [comment]
Ah, there's the problem.  The button presumably works fine.  Go all the way down to the bottom of README.md.  

```
* Argo Slack:   [click here to join](https://join.slack.com/t/argoproj/shared_invite/enQtMzExODU3MzIyNjYzLWUxZDYyODIyYzY3N2RjOWMyNDA4NmFjMTNkMTE1ODI2OGY3MzgyMWFmMmY3N2UzNWRmOWFmMGY4NTBhZGQxYWY)
```

**Comment by athornton** [comment]
#1710

**Comment by simster7** [comment]
Closed via #1710
